src/ucs/config/parser.c

```diff
@@ -855,6 +858,84 @@ void ucs_config_help_array(char *buf, size_t max, const void *arg)
     array->parser.help(buf + strlen(buf), max - strlen(buf), array->parser.arg);
 }
 
+int ucs_config_sscanf_allow_list(const char *buf, void *dest, const void *arg)
+{
+    ucs_config_allow_list_t *field  = dest;
+    unsigned offset                 = 0;
+
+    if (buf != NULL && buf[0] == ucs_config_parser_negate) {
```

Comment:
1. pls add ( ) around != and == expressions
2. we can assume buf!=NULL, linke sscanf() does for example

Reply:
2 - If I recall correctly, not having this check failed the pipeline. Will remove it as retest.